runtime/include/vendor_version.h

```diff
@@ -19,10 +19,10 @@
  *
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
  *******************************************************************************/
-
-/* Example usage for repo and sha.  This value will be inserted into the
- * java.fullversion and java.vm.info system properties
- * #define VENDOR_VERSION_STRING "<repo name> - <sha>"
- * or descriptive string
- * #define VENDOR_VERSION_STRING "Add some experimental code to java.base"
+/* Example usage for inclusion of a vendor name and repository sha.  These values
+ * will be inserted into the java.fullversion and java.vm.info system properties
+ *
+ * #define VENDOR_NAME		"ABC Widgets Inc."
+ * #define VENDOR_SHORT_NAME	"ABC"
+ * #define VENDOR_SHA		"1a2b3c4"
```

Comment:
Please don't use internal tabs, as the values don't line up with all tab settings.